src/components/brush.js

```diff
@@ -65,7 +68,14 @@ AFRAME.registerComponent('brush', {
       }
     });
   },
+
+  triggerModeChanged:  function() {
+    this.enabled = this.data.enabled && this.el.getAttribute('trigger-mode') == 'brush'
```

Comment:
Is the proper way to do this to have another component that manages the trigger-mode attribute to data.enabled binding?  To make this more reusable it would be nice if this didn't have to worry about the external trigger-mode attribute